Common/Data/Auxiliary/LocalDiskFactorFileProvider.cs

```diff
@@ -71,7 +72,7 @@ public FactorFile Get(Symbol symbol)
                 return GetFactorFile(symbol, symbol.Value, market);
             }
 
-            var mapFile = mapFileResolver.ResolveMapFile(symbol.Value, symbol.ID.Date);
+            var mapFile = mapFileResolver.ResolveMapFile(symbol.Value, DateTime.Today);
```

Comment:
If the map files are always using today's ticker string isn't this right? (symbol.Value is today's + the date of today?)